packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -140,7 +147,7 @@ export function TimeRangePicker(props: TimeRangePickerProps) {
 
       {hasAbsolute && (
         <ToolbarButton
-          aria-label="Move time range forwards"
+          aria-label={t('time-picker.range-picker.move-time-forwards', 'Move time range forwards')}
```

Comment:
Question: would it be a good idea to include `aria` or `aria-label` in the key to make it more explicit? In the long run we will probably have someone translating these text that probably won't look in the code. They will most likely view the UI and try to use the key to understand what part of the UI the text relates to. When it comes to aria-labels they are not visible in the UI so I'm thinking that they would probably need to spend a lot of time figuring this out without any indication in the key it self.

I saw that you had a similar case with the `tooltip` where you added that in the end which I really like!

Reply:
Hi @mckn! First of all, thanks for your feedback. 
Following your suggestions, I changed the key to `time-picker.range-picker.forwards-time-aria-label` and `time-picker.range-picker.backwards-time-aria-label`.

Reply:
I think this is a good question @mckn. I'm sympathetic towards your reasoning (to include more context in the key for translators), but my gut feel is that we shouldn't do this because it creates a higher opportunity for a bifurcated experience for those that use the aria-label content vs those who don't.

I don't have a particularly strong opinion here either way, but I don't want us to immediately jump to this.